pkg/ovs/ovsconfig/ovs_client.go

```diff
@@ -694,6 +695,17 @@ func buildPortDataCommon(port, intf map[string]interface{}, portData *OVSPortDat
 	} else { // ofport not assigned by OVS yet
 		portData.OFPort = 0
 	}
+	var macInUse string
+	if field, ok := intf["mac_in_use"].(string); ok {
+		macInUse = field
+	} else if fields, ok := intf["mac_in_use"].([]interface{}); ok {
```

Comment:
how could this happen? The type of the field is not deterministic?

Reply:
This is found in the integration test, it looks the "system" typed Interface uses this format.